JSDemos/Demos/Charts/SimpleArray/description.md

```diff
@@ -1 +1,3 @@
-In&nbsp;this demo, the Chart component is&nbsp;populated with data taken from a&nbsp;simple JavaScript array. It&nbsp;is&nbsp;the easiest way to&nbsp;provide data for the component, and it&nbsp;is&nbsp;the right choice if&nbsp;you do&nbsp;not need to&nbsp;get data from the server or&nbsp;update data dynamically.
\ No newline at end of file
+You can populate the Chart component with data locally or remotely. If you do not need to get data from the server or update data dynamically, bind the component to a local JavaScript array. To bind Chart to a local array, pass this array to the Chart's [dataSource](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#dataSource) property. 
```

Comment:
First you give the reader two choices: local or remote data. Then you repeat the remote option (server), but then introduce a whole new concept (update dynamically), and then talk about local data (array) again. So in the first sentence, I had two options. IN the second, I also see a third - dynamic data... But I don't know if it relates to remote data, or local, or both. So I suggest that we combine all of that.

```suggestion
If you do not need to populate the chart with data from a remote server, and the chart doesn't need to update data frequently, then simply bind the component to a local JavaScript array. Pass the array to the Chart's [dataSource](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#dataSource) property. 
```

Reply:
Can you please explain to me when we start the component's name with the capital letter, and when we use only lowercase letters? I thought that when we are talking about the component, then we use the capital letter, e.g. the Chart component. And when we talk about the chart as a visual element in general, not necessarily the DX component, we use only the lowercase.
I thought that I'm writing here about our component specifically, because it's not some generic chart, but the Chart that can be populated with data in specific ways. And later, when I'm talking about the dataSource, I also write "the Chart's dataSource", since the dataSource property belongs to the component. 
I just think it's not consistent to write about the component "Chart/chart" in one rather small text.

Reply:
"Chart" is fine. Sorry for the confusion :-) I didn't really mean to change that. 

In general, I'd say it's fine to use lowercase "chart" as a synonym, if the reader knows by now what kind of chart we're talking about. That could help avoid repetition if, say, your control's name is ASPxChartControl. That way you can alternate that with a simple word "chart", and make text a bit easier to read. 

But if your control's name is already simple enough ("Chart"), it doesn't really make sense to use a lowercase "synonym" :-) You're right.  

```suggestion
If you do not need to populate the Chart with data from a remote server, and the data doesn't change frequently, then simply bind the component to a local JavaScript array. Pass the array to the Chart's [dataSource](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#dataSource) property. 
```